### DISCUSSION TITLE
Close Discussions about too old versions?

### DISCUSSION THREAD

**Discussion by inson1** [clementwzk/OpenCalc#357] [Ideas]
Lets say there should be last 5 opened stable versions Discussions + all betas? More than that is maybe too much?
What do you think @Darkempire78?

**Comment by clementwzk** [comment]
I'm not sure that discussions about a specific version are usefull. That's why I didn't open one for the latest release

**Reply by inson1** [reply]
@Darkempire78 well, that is up to you. I'm just saying that the old ones arent too much usefull. And maybe should be closed. Its little bit cluttering the discussions tbh.

**Comment by clementwzk** [comment]
done

**Reply by inson1** [reply]
@Darkempire78 What about #165 ?
